src/main/java/plansoft/gestionecompetenze/dipendente/Dipendente.java

```diff
@@ -0,0 +1,44 @@
+package plansoft.gestionecompetenze.dipendente;
+
+import javax.persistence.Entity;
+import javax.persistence.Id;
+
+@Entity
+public class Dipendente {
```

Comment:
Nonostante la nostra lingua madre sia l'italiano, dobbiamo cercare di scrivere il codice in lingua inglese.

Dipendenti = Employees